### ISSUE TITLE
`AnyAttrAllowed` missing from django-stubs-ext

### ISSUE THREAD

**Issue by g-as** [typeddjango/django-stubs#892]
# Bug report

## What's wrong

```
    annotated_type = get_or_create_annotated_type(api, model_type, fields_dict=fields_dict)
  File "/home/backend/.venv/lib/python3.9/site-packages/mypy_django_plugin/transformers/models.py", line 529, in get_or_create_annotated_type
    annotated_model_type = api.named_generic_type(ANY_ATTR_ALLOWED_CLASS_FULLNAME, [])
  File "/home/backend/.venv/lib/python3.9/site-packages/mypy/checker.py", line 5287, in named_generic_type
    info = self.lookup_typeinfo(name)
  File "/home/backend/.venv/lib/python3.9/site-packages/mypy/checker.py", line 5294, in lookup_typeinfo
    sym = self.lookup_qualified(fullname)
  File "/home/backend/.venv/lib/python3.9/site-packages/mypy/checker.py", line 5361, in lookup_qualified
    raise KeyError(msg.format(last, name))
KeyError: "Failed qualified lookup: 'AnyAttrAllowed' (fullname = 'django_stubs_ext.AnyAttrAllowed')."
```

## How is that should be

I feel there's a mismatch between django-stubs and django-stubs-ext following #753, where a new version of django-stubs-ext containing `AnyAttrAllowed` has not been released.

## System information

- OS:
- `python` version: 3.9
- `django` version: 3.2
- `mypy` version: 0.942
- `django-stubs` version: 1.10
- `django-stubs-ext` version: 0.3.1

**Comment by FerenetsRoman** [comment]
Have the same issue. Could you please make a release for `django-stubs-ext` to fix it?

**Comment by g-as** [comment]
Fixed by 4e3f9d6f06d6253f28638a52b795ad00569fe147
